pkg/continuoustest/client.go

```diff
@@ -58,7 +59,7 @@ type ClientConfig struct {
 }
 
 func (cfg *ClientConfig) RegisterFlags(f *flag.FlagSet) {
-	f.StringVar(&cfg.TenantID, "tests.tenant-id", "anonymous", "The tenant ID to use to write and read metrics in tests. (mutually exclusive with basic-auth or bearer-token flags)")
+	f.StringVar(&cfg.TenantID, "tests.tenant-id", defaultTenant, "The tenant ID to use to write and read metrics in tests. (mutually exclusive with basic-auth or bearer-token flags)")
```

Comment:
This is no longer mutually exclusive with basic auth.

Reply:
👍
Also had a peek at the continuous test docs, doesn't look like there is anything to update there.